packages/core/permissions/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "tsconfig/base.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "include": ["types", "src"],
```

Comment:
I know this is a copy paste from other packages, but since we don't have a types/ folder, do we want to keep this & anticipate the day we'll have one? Or do we want to only include stuff that's actually there?